src/transformers/modeling_utils.py

```diff
@@ -3328,6 +3328,9 @@ def from_pretrained(
             elif metadata.get("format") == "flax":
                 from_flax = True
                 logger.info("A Flax safetensors file is being loaded in a PyTorch model.")
+            elif metadata.get("format") == "mlx":
+                # This is a mlx file, we assume weights are compatible with pt
+                pass
```

Comment:
This works, but it goes down a code path that requires `mlx` to be installed via https://github.com/huggingface/transformers/blob/63caa370e6c618dbe7d3fd4cbf545cc32eca1a15/src/transformers/utils/generic.py#L118.

I think a better solution in this case could be to replace the `format` with `pt`.

Reply:
I was mistaken, `mlx` is only used if installed, otherwise it works fine.